### PR TITLE
Create some methods to get metrics data from opemmanage

### PR DESCRIPTION
**Pull request by Pengpengwanga**
Signed-off-by: Pengpeng Wang <pengpengw@vmware.com>

### CODE REVIEW

File: flowgate-common/src/main/java/com/vmware/flowgate/common/MetricName.java [hunk 1]
```diff
@@ -33,6 +33,13 @@
    public static final String SERVER_CONNECTED_PDU_POWER = "PduPower";
    public static final String SERVER_USED_PDU_OUTLET_CURRENT = "ConnectedPDUOutletCurrent";
    public static final String SERVER_USED_PDU_OUTLET_POWER = "ConnectedPDUOutletPower";
+   public static final String SERVER_ENERGY_CONSUMPTION = "EnergyConsumption";//(KWH)
```
Comment (yixingjia) [inline flowgate-common/src/main/java/com/vmware/flowgate/common/MetricName.java:36]: Let's use full metric name.

File: openmanage/src/main/java/com/vmware/flowgate/openmanage/client/OpenManageAPIClient.java [hunk 2]
```diff
@@ -97,6 +100,43 @@ public void getToken() {
...
+      ResponseEntity<DeviceTemperature> temperatureEntity = this.restTemplate.exchange(
+            getServiceEndPoint() + String.format(GetDeviceTemperatureUri, deviceId), HttpMethod.GET,
+            getDefaultEntity(), DeviceTemperature.class);
+      if(temperatureEntity.hasBody()) {
+         return temperatureEntity.getBody();
+      }
+      return null;
+   }
+
+   public boolean enabledPowerManagePlugin() {
```
Comment (yixingjia) [inline openmanage/src/main/java/com/vmware/flowgate/openmanage/client/OpenManageAPIClient.java:123]: isPowerManagePluginEnabled

File: flowgate-common/src/main/java/com/vmware/flowgate/common/MetricName.java (hunk 1 above)
Comment (Pengpengwanga) [reply flowgate-common/src/main/java/com/vmware/flowgate/common/MetricName.java:36]: Done

File: openmanage/src/main/java/com/vmware/flowgate/openmanage/client/OpenManageAPIClient.java (hunk 2 above)
Comment (Pengpengwanga) [reply openmanage/src/main/java/com/vmware/flowgate/openmanage/client/OpenManageAPIClient.java]: Done

**Review by yixingjia — APPROVED** [review]

**Pull request merged**